packages/applications-service-api/src/controllers/advice.js

```diff
@@ -40,5 +40,30 @@ module.exports = {
 			logger.error(e.message);
 			res.status(StatusCodes.INTERNAL_SERVER_ERROR).send(`Problem getting advice \n ${e}`);
 		}
+	},
+
+	async getAdviceById(req, res) {
+		const { adviceID } = req.params;
+		try {
+			logger.debug(`Retrieving advice by ID...`);
+
+			const advice = await getAdviceById(adviceID);
+
+			if (!advice) {
+				throw ApiError.adviceNotFound(adviceID);
+			}
+
+			res.status(StatusCodes.OK).send(advice);
+		} catch (e) {
+			if (e instanceof ApiError) {
+				logger.debug(e.message);
+				res.status(e.code).send({ code: e.code, errors: e.message.errors });
+				return;
+			}
+			logger.error(e.message);
+			res
+				.status(StatusCodes.INTERNAL_SERVER_ERROR)
+				.send(`Problem getting advice ${adviceID} \n ${e}`);
```

Comment:
we might want to send a JSON object here, i think the web-app will try and parse this as json and fail. This might be the case on other controllers too so we can address this as a tech debt ticket if it's a big change

Reply:
I think the webapp just looks at the 500 status message - but yes, makes sense to have it in JSON if we are bothering to send it. 👍

Reply:
Updated on these new endpoints